### DISCUSSION TITLE
Need Datacamp subscription

### DISCUSSION THREAD

**Discussion by abdullahh01** [community/community#49023] [GitHub Education]
![Screenshot_20230303-004828](https://user-images.githubusercontent.com/118959432/222536566-ee483ac9-b51e-436e-938f-f257dd528f82.jpg)
![Screenshot_20230303-004821](https://user-images.githubusercontent.com/118959432/222536604-f18be91e-77ac-4859-bacb-6f1bd51befe2.jpg)
Hello I am a Computer Science Student and i want datacamp subscription because i want to do more courses of data science and my subscription as been cancelled and i have done 2 to 3 course's from DataCamp sir i am unemployed and an student so plz give me the opportunity

**Comment by samus-aran** [comment]
Hi @abdullahh01 👋 

Unfortunately we can not extend your free subscription for Datacamp. The Global Campus Benefits provides free 3-month Datacamp individual subscription for students. We understand that this news may not what you were hoping for. We hope you find more benefits in the other offers within the Global Campus benefits.
